public/js/app.js

```diff
@@ -19,6 +19,61 @@ jQuery( document ).ready(function( $ ){
                 $('#myEdit').modal('show');
             });
         });
+        $('.comment-btn').on('click', makeComment);
     }
 );
 
+/**
+ * Allows users make comment without modal reloading
+ */
+var makeComment = function(evt){
+  evt.preventDefault();
+  var _this = $(this);
+  var comment = _this.parent().find('textarea');
+  var form = _this.parent().parent();
+  var _token = $('input[name=_token]').val();
+  if(comment){
+    $.post(form.attr('action'), {'comment':comment.val(), '_token': _token})
+    .success(
+        function(respData){
+          var newComment =
+              '<div class="comment-wrapper">'
+            + '<img src="'+respData.avatar+'" class="comment-img"/>'
+            + '<span class="comment-username">'
+            + '<a href="#" class="no-decoration">'
+            + respData.username
+            + '</a>'
+            + '</span>'
+            + '<div class="comment-comment">'
+            + respData.comment
+            + '</div>'
+            + '<div class="comment-time">'
+            + respData.comment_time
```

Comment:
Bad line breaking before '+'.<br>Identifier 'comment_time' is not in camel case.